script.js

```diff
@@ -0,0 +1,90 @@
+var trevor = {
+  name: 'Trevor',
+  position: {
+    x: 0,
+    y: 0
+  },
+  health: 100,
+  mana: 100,
+  burdon: 28,
+  strength: 5,
+  constitution: 12,
+  magic: 10,
+  spell: {
+    name: "Fireball",
+    strength: 10
+  },
+  weapon: undefined,
+
+  yell() {
+    console.log("LEEEROOYYYYYY JENNNKKKKIINNNNNSSSS!!!!!");
+  }
+};
+
+var adam = {
+  name: 'Adam',
+  health: 100,
+  mana: 100,
+  burdon: 40,
+  constitution: 14,
+  strength: 5,
+  spell: undefined,
+  weapon: {
+    name: "Lion's Roar",
+    type: "broadsword",
+    strength: 8,
+  },
+  yell() {
+    console.log('I FIGHT FOR MY FRIENDS!!');
+  }
+};
+
+function takeDamage(ammount, target) {
```

Comment:
It doesn't matter since `ammount` is used throughout but just because it's me and spelling: `amount`. 😄